ldm/simplet2i.py

```diff
@@ -323,6 +336,21 @@ def process_image(image,seed):
                             image_callback(image, seed)
                     seed = self._new_seed()
 
+                if gfpgan_strength > 0:
```

Comment:
This never gets run

Reply:
Are you passing GFPGAN as a prompt argument?

`<prompt> -G 0.5`